omg/core/views.py

```diff
@@ -108,8 +109,21 @@ def setup(self):
         )
 
         # Pickup button image
-        self.pickup_button_image = os.path.join(ASSET_DIR, "pickup_button", "F.png")
-        self.pickup_button_image_scale = 0.1
+        # Set grey_background
+        self.pickup_grey_background = os.path.join(
+            ASSET_DIR, "pickup_button", "grey_background.png"
+        )
+        self.pickup_grey_background_image_scale: float = 0.3
+
+        # Set button text attributes
+        pickup_key_text: str = "F"
```

Comment:
I think we can get rid of hard-coding here. If you notice, arcade.key buttons are mapped according to the ASCII table. For example, f is 102, g is 103 etc. So if you convert numeric value to the letter, you can adjust to whatever button the player has set for the pickup button (at least true for the number from 0 to 9 and letters in the alphabet, which would cover the sufficient amount of cases for our game)